app/scripts/modules/google/src/loadBalancer/configure/internalhttp/createInternalHttpLoadBalancer.controller.js

```diff
@@ -0,0 +1,162 @@
+'use strict';
```

Comment:
All new files should be added in Typescript; please convert this file to TS.

Reply:
OK, I'll fix it.